oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/web/DefaultBearerTokenResolverTests.java

```diff
@@ -51,6 +51,24 @@ public void resolveWhenValidHeaderIsPresentThenTokenIsResolved() {
 		assertThat(this.resolver.resolve(request)).isEqualTo(TEST_TOKEN);
 	}
 
+	@Test
+	public void resolveWhenValidHeaderIsPresentWithSingleBytePaddingIndicatorThenTokenIsResolved() {
+		String token = TEST_TOKEN + "=";
```

Comment:
I think we could get some extra coverage by changing this one to check a different way to extract the token. For example, you might change this to `resolveWhenQueryParameterEndsWithPaddingThenTokenIsResolved`.